scripts/check-dependencies.js

```diff
@@ -58,7 +58,10 @@ const commonConfigs = [
 // We are going to copy these devDependencies into each package from this script,
 // in order to standardize them.
 const commonDevDeps = {
-    'cross-env': '^5.2.0',
+    'cross-env': require('../packages/internal-lib-build/package.json').dependencies['cross-env'],
+}
+if (!commonDevDeps['cross-env']) {
+    throw new Error('Expected `internal-lib-build` to have `cross-env` as a dependency. You should revisit the assumptions in `./scripts/check-dependencies.js`.')
```

Comment:
Decided to "pin" the common version so we don't have to update this file every time we update that one particular dep.